# request params
sensor_msgs/Image rgb_image
---
# response params
BoundingBoxes bounding_boxes
